glusterfs/roles/glusterfs_server/tasks/130_gluster_add_bricks.yaml

```diff
@@ -1,18 +0,0 @@
-- set_fact:
-    cluster_hosts: "{{ play_hosts | join(',') }}"
-
-- debug: var=cluster_hosts
-
-- name: create gluster volume
-  gluster_volume:
-    state=present
-    name=elastic_backups
-    bricks="{{ gluster_storage_dir }}"
-    cluster="{{ cluster_hosts }}"
-    force=yes
-    rebalance=no
-    transport=tcp
-    replicas={{ num_hosts  }}
```

Comment:
Number of replicas was changed with explicit replica_factor=3, because more, than 3 replicas - is overhead and not recommended by glusterfs design